File: src/main/kotlin/com/study/biddingwar/Item.kt
package com.study.biddingwar

import javax.persistence.* // Spring Data JPA(Java Persistance API)
import java.time.LocalDateTime

@Entity
class Item(
    @Id
    @GeneratedValue
    var id: Long? = null,

    var seller: String,

    var title: String,

    var category: String,

    var initialPrice: Long = 1000,

    var description: String? = "",

    var status: String = "registered",

    var registered: LocalDateTime = LocalDateTime.now(),

    var finished: LocalDateTime? = null,
)

File: front/src/components/Item.js
const Item = ({ item }) => {
    return (
        <div className='item'>
            <h3>{item.title}</h3>
            <ul>
                <li>시작가: {item.initialPrice}</li>
                <li>설명: {item.description}</li>
                <li>등록일: {item.registeredDate}</li>
            </ul>
        </div>
    )
}

export default Item
